physionet-django/console/urls.py

```diff
@@ -92,6 +93,8 @@
 
     # guidelines
     path('guidelines/review/', views.guidelines_review, name='guidelines_review'),
+    path('guidelines/course/', views.guidelines_course, name='guidelines_course'),
```

Comment:
This page is not currently listed in the admin console (not sure if this is intentional).